api/src/main/java/marquez/cli/DbRetentionCommand.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2018-2023 contributors to the Marquez project
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package marquez.cli;
+
+import io.dropwizard.cli.ConfiguredCommand;
+import io.dropwizard.db.DataSourceFactory;
+import io.dropwizard.db.ManagedDataSource;
+import io.dropwizard.setup.Bootstrap;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import marquez.MarquezConfig;
+import marquez.db.DbRetention;
+import marquez.db.exceptions.DbRetentionException;
+import net.sourceforge.argparse4j.inf.Namespace;
+import org.jdbi.v3.core.Jdbi;
+import org.jdbi.v3.postgres.PostgresPlugin;
+
+/**
+ * A command to apply retention policy to source, dataset, and job metadata collected by Marquez.
+ *
+ * <h2>Usage</h2>
+ *
+ * For example, to override the {@code retention-days}:
+ *
+ * <pre>{@code
+ * java -jar marquez-api.jar db-retention --retention-days 14
+ * }</pre>
+ */
+@Slf4j
+public class DbRetentionCommand extends ConfiguredCommand<MarquezConfig> {
+  private static final String DB_SOURCE_NAME = "db-retention-source";
+
+  /* Default retain for metadata. */
+  private static final int DEFAULT_RETENTION_DAYS = 7;
```

Comment:
Shouldn't we use the same constants for the default retention in config and on the command line param?

Reply:
Absolutely! So ok, got a bit too carried away with all these constants 😅